_layouts/default.html

```diff
@@ -41,7 +41,5 @@
 <!-- /.container -->
     </div>
 </body>
-{% if site.google_analytics %}
-{% include google_analytics.html %}
-{% endif %}
+
```

Comment:
Not exactly sure how it works, but why do we not replace `google_analytics` with the new `analytics` file?

Reply:
Because this layout loads the `head.html` ... which in turn loads `analytics.html`